indra/newview/llviewerobject.cpp

```diff
@@ -4820,6 +4820,18 @@ void LLViewerObject::setPositionParent(const LLVector3 &pos_parent, bool damped)
     else
     {
         setPositionRegion(pos_parent, damped);
+
+        // #1964 mark reflection probe in the linkset to update position after moving via script
+        for (LLViewerObject* child : mChildList)
+        {
+            if (child && child->isReflectionProbe())
+            {
+                if (LLDrawable* drawablep = child->mDrawable)
+                {
+                    gPipeline.markMoved(drawablep);
```

Comment:
Should viewer be the one doing this? Shouldn't server inform viewer about the move?

Reply:
I'm not sure, maybe. Anyway, we may remake this quickfix for probes later.